### PR TITLE
Fix(loginCode): correct timer in login code

### PR DESCRIPTION
**Pull request by sahar-shafiee**

### CODE REVIEW

**Comment by sahar-shafiee** [comment]
Task linked: [CU-27h6p7r add timer in login code page](https://app.clickup.com/t/27h6p7r)

**Comment by likecodingloveproblems** [comment]
It has both styling and functionality changes :cry:

File: containers/login/timer.jsx [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
 function AppTimer({ timer = 59, setTimer }) {
-  //   const [timer, setTimer] = useState(time);
 
   useEffect(() => {
-    const timing =
-      timer > 0 &&
-      setInterval(() => {
-        setTimer(timer - 1);
-        localStorage.setItem("timing", timer - 1);
-      }, 1000);
```
Comment (likecodingloveproblems) [inline containers/login/timer.jsx:14]: we need timer > 0 predicate

File: containers/login/timer.jsx [hunk 2]
```diff
@@ -1,32 +1,24 @@
...
+    const timing = setInterval(() => {
+      setTimer(timer - 1);
+      localStorage.setItem("timing", timer - 1);
+    }, 1000);
```
Comment (likecodingloveproblems) [inline containers/login/timer.jsx:13]:
``` suggestion 
    const timing = timer > 0 && setInterval(() => {
      setTimer(timer - 1);
      localStorage.setItem("timing", timer - 1);
    }, 1000);
```

**Review by likecodingloveproblems — APPROVED** [review]

**Pull request merged**